api/routers/feature.py

```diff
@@ -0,0 +1,140 @@
+import json
+from typing import Annotated
+
+import datastore_pb2 as dstore
+import formatters
+from fastapi import APIRouter
+from fastapi import HTTPException
+from fastapi import Path
+from fastapi import Query
+from geojson_pydantic import Feature
+from geojson_pydantic import FeatureCollection
+from grpc_getter import get_obs_request
+from grpc_getter import get_spatial_extent
+from jinja2 import Environment
+from jinja2 import FileSystemLoader
+from jinja2 import select_autoescape
+from shapely import geometry
+from utilities import get_datetime_range
+
+router = APIRouter(prefix="/collections/observations")
+
+env = Environment(loader=FileSystemLoader("templates"), autoescape=select_autoescape())
+
+
+@router.get(
+    "/items", tags=["Collection items"], response_model=Feature | FeatureCollection, response_model_exclude_none=True
+)
+async def search_timeseries(
+    bbox: Annotated[str | None, Query(example="5.0,52.0,6.0,52.1")] = None,
+    datetime: Annotated[
+        str | None,
+        Query(
+            example="2022-12-31T00:00Z/2023-01-01T00:00Z",
+            description="E-SOH database only contains data from the last 24 hours",
+        ),
+    ] = None,
+    ids: Annotated[str | None, Query(description="Comma separated list of time series ids")] = None,
+    parameter_name: Annotated[str | None, Query(alias="parameter-name", description="E-SOH parameter name")] = None,
```

Comment:
All of the parameters support comma seperated lists. Do we need to mention this for each one?

Reply:
No, will remove.